packages/front/src/components/Claim/ClaimSentence.css

```diff
@@ -1,15 +1,7 @@
-button.sentence {
+a.sentence {
```

Comment:
Maybe instead of specifying the element (`button`, `a`), remove the element and use only the class name, to reduce specificity. And use a semantic name for the class to you know that this class is about a button, like `.btn-sentence`, `.button-sentence`.